website/docs/pushdown.md

```diff
@@ -17,6 +17,18 @@ To make this process more efficient, we minimize the amount of incoming data, by
 You can learn more about query pushdown in our [blog post](https://blog.ferretdb.io/ferretdb-fetches-data-query-pushdown/).
 :::
 
+## Types of pushdown
+
+Currently we support 3 different types of pushdown.
+
+The **Filter pushdown**, which is enabled by default, sends supported filters directly to the underlying database.
+It can be disabled with `--disable-filter-pushdown`.
+The **Sort Pushdown**, which can be enabled with `--enable-sort-pushdown`, pushes down sorting, `limit`, and `skip` methods.
```

Comment:
Hmm, what I don't understand is the difference between "safe" and "unsafe" sort pushdown... If we have a "safe" sort pushdown, why don't we enable it by default?

Reply:
No longer part of this PR